src/transform/FooterReadMore.js

```diff
@@ -0,0 +1,349 @@
+import './FooterReadMore.css'
+
+/**
+ * @typedef {function} SaveButtonClickHandler
+ * @param {!string} title
+ * @return {void}
+ */
+
+/**
+  * @typedef {function} TitlesShownHandler
+  * @param {!string[]} titles
+  * @return {void}
+  */
+
+/**
+   * Display fetched read more pages.
+   * @typedef {function} ShownReadMoreHandler
+   * @param {!Object[]} pages
+   * @param {!Document} document
+   * @param {!string} containerID
+   * @param {SaveButtonClickHandler} saveButtonClickHandler
+   * @param {TitlesShownHandler} titlesShownHandler
+   * @return {void}
+   */
+
+let _saveForLaterString = null
```

Comment:
These globals suggest that a class implementation may be more appropriate. The client can than keep one or more instances of that class as needed.